packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -118,7 +118,7 @@ class CameraValue {
   /// Whether setting the focus point is supported.
   final bool focusPointSupported;
 
-  /// The current device orientation.
+  /// The current device UI orientation.
```

Comment:
what does UI refer to?

Reply:
User Interface. The previous implementation always reflected the physical device orientation, regardless what the UI orientation actually is. The new implementation always reflects the UI orientation instead.